Anton.Konovalov/Homework2/brackets.rb

```diff
@@ -0,0 +1,21 @@
+class String
```

Comment:
Hey, sorry, but this method isn't working for valid cases like
"()[]{}"
Your method returns false while all the brackets are correctly opened and closed.